lang/ui.ca.json

```diff
@@ -0,0 +1,1161 @@
+  },
+  "post-save-title": {
+    "defaultMessage": "Projecte desat",
+    "description": "Title of dialog shown after the user saves the project as a hex file."
+  },
+  "post-save-transfer-hex": {
+    "defaultMessage": "Per executar aquest fitxer hexadecimal a la teva micro:bit segueix aquests passos <link>follow these steps</link>.",
+    "description": "Message in dialog shown after the user saves the project as a hex file."
+  },
+  "project-actions": {
+    "defaultMessage": "Accions del projecte",
+    "description": "Aria label for the bar with project actions"
+  },
+  "project-header": {
+    "defaultMessage": "Capçalera del projecte",
+    "description": "Aria label for the project header area"
+  },
+  "project-name": {
+    "defaultMessage": "Nom del projecte",
+    "description": "Text used to indicate the project name"
+  },
+  "project-name-not-empty": {
+    "defaultMessage": "El nom del projecte no pot estar buit",
+    "description": "Validation message for project name"
+  },
+  "project-tab": {
+    "defaultMessage": "Projecte",
+    "description": "Project tab button text"
+  },
+  "project-tab-description": {
+    "defaultMessage": "Visualitza, crea, afegeix i edita els fitxers del teu projecte",
+    "description": "Project tab description"
+  },
+  "python-powered": {
+    "defaultMessage": "Fet amb Python",
+    "description": "Python powered logo alt text"
+  },
+  "python-tab": {
+    "defaultMessage": "Python",
+    "description": "Python tab text"
+  },
+  "quit-anyway": {
+    "defaultMessage": "Alguns dels teus canvis no s'han desat. Sortir de totes maneres?",
+    "description": "Quit anyway text"
+  },
+  "read-less": {
+    "defaultMessage": "Llegir menys",
+    "description": "Action text to collapse an expanded section"
+  },
+  "read-more": {
+    "defaultMessage": "Més informació",
+    "description": "Action text to expand a collapsed section"
+  },
+  "redo": {
+    "defaultMessage": "Refer (redo)",
+    "description": "Aria label for the redo button"
+  },
+  "reference-tab": {
+    "defaultMessage": "Referència",
+    "description": "Reference tab button text"
+  },
+  "replace-action-label": {
+    "defaultMessage": "Substitueix",
+    "description": "Action label for replacing project dialog"
+  },
+  "reset-project-action": {
+    "defaultMessage": "Restableix el projecte",
+    "description": "Action to reset the project to its default state"
+  },
+  "reset-project-feedback": {
+    "defaultMessage": "Restableix el projecte al programa d'inici predeterminat",
+    "description": "Confirmation message after resetting the project"
+  },
+  "reset-project-hover": {
+    "defaultMessage": "Restableix el projecte al programa inicial predeterminat, descartant el teu treball",
+    "description": "Reset action hover text"
+  },
+  "results-count": {
+    "defaultMessage": "{count, plural, =0 {No results} one {# result} other {# results}}",
```

Comment:
I doubt this is Catalan.
Crowdin's UI allows the user to enter a number and see the string for each case. See the link in the description for the ICU format.
I think review of all ICU format strings is prudent as there are others that aren't translated where it was less clear to me whether they should be.

Reply:
Requested in Crowdin